src/Symfony/Bridge/Twig/Tests/AppVariableTest.php

```diff
@@ -212,6 +212,11 @@ public function testGetFlashes()
             array('notice' => $flashMessages['notice'], 'error' => $flashMessages['error']),
             $this->appVariable->getFlashes(array('notice', 'error'))
         );
+
+        $this->assertEquals(
+            array('warning' => $flashMessages['warning']),
+            $this->appVariable->getFlashes(array('warning'))
```

Comment:
I wanted to add a help note in this test, but your PR doesn't allow changes. Here is what I'd like to add to avoid confusing tests:

```php
$this->assertEquals(
    array('warning' => $flashMessages['warning']),
    $this->appVariable->getFlashes(array('warning')),
    'After getting some flash types (e.g. "notice" and "error"), the rest of flash messages must remain (e.g. "warning").'
);
```